.github/workflows/release-test.yml

```diff
@@ -181,6 +181,25 @@ jobs:
         mkdir test-reports/
         junit2html ${REPORT_XML} ${REPORT_NAME}.html
         cp ${REPORT_XML} ${REPORT_NAME}.xml
+    - name: Generate result message
+      if: always()
+      id: generate_results
+      run: |
+        if [ "${{ steps.tests.conclusion }}" == "success" ]; then
+          nice_str="&#x2705; <b>PASSED</b>"
+        elif [ "${{ steps.tests.conclusion }}" == "failure" ]; then
+          nice_str="&#x274C; <b>FAILED</b>"
+        fi
+        echo "nice_str=${nice_str}" >> ${GITHUB_OUTPUT}
+    - name: Report to Matrix channel
```

Comment:
May I suggest a pre-made action for this? We're using one for RIOT-RS ([link to workflow](https://github.com/future-proof-iot/RIOT-rs/blob/main/.github/workflows/matrix-bot.yml).

Sending a message then turns into:
```
     - name: send message
       uses: s3krit/matrix-message-action@v0.0.3
       with:
         room_id: ${{ secrets.MATRIX_ROOM_ID }}
         access_token: ${{ secrets.MATRIX_ACCESS_TOKEN }}
         message: "PR merged: [${{ github.event.pull_request.title }}](${{ github.event.pull_request.html_url }})"
         server: "matrix.org"

Reply:
That's the one I discarded in https://github.com/RIOT-OS/RIOT/pull/19879. It did not work.

Reply:
Wasn't aware that you discarded an action before. But, that was "uses: fadenb/matrix-chat-message@v0.0.6", which is a different one. The one I linked works fine, I just got another notification in the RIOT-rs channel.

Reply:
Ah since the user name ringed a bell (but I tink from a different context), I assumed it to be the same ^^"